pkg/bpm/bpm.go

```diff
@@ -301,6 +315,26 @@ func (m *BackgroundProcessManager) Stdio(pid int, startTime int64) *Stdio {
 	return io.(*Stdio)
 }
 
+// GetControlledProcessIDs find all process IDs controlled by BPM
+func (m *BackgroundProcessManager) GetControlledProcessIDs() ([]int, error) {
+	processes, err := ps.Processes()
+	if err != nil {
+		log.Error(err, "fail to get current process processes")
+		return nil, err
+	}
+
+	var res []int
+	pid := os.Getpid()
+	for _, p := range processes {
+		if pid == p.PPid() {
+			res = append(res, p.Pid())
+		}
+	}
+
+	log.Info("all controlled processes has been scanned", "PID", res)
+	return res, nil
+}
```

Comment:
This implementation may introduce more processes that might be not controller by this bpm instance. You are finding that all the processes that controller by **this process**, but one process might use more than one instance of `bpm` and has other logic that spawns new processes. 

I suggest that use `identifiers *sync.Map` in struct `BackgroundProcessManager`, the key of this map store the pid.

Reply:
OK, I will modify it later.

Reply:
Done.